homeassistant/components/modbus/sensor.py

```diff
@@ -121,6 +122,9 @@ async def async_setup_platform(
             del entry[CONF_REGISTER_TYPE]
 
     for entry in discovery_info[CONF_SENSORS]:
+
+        regSize = entry[CONF_COUNT] * 2 * entry.get(CONF_REGISTER_SIZE, 1)
+
         if entry[CONF_DATA_TYPE] == DATA_TYPE_STRING:
             structure = str(entry[CONF_COUNT] * 2) + "s"
```

Comment:
Please remove empty line, and I can see line 129 have not been corrected (independent of multiplier or size).